src/shapes/group.class.js

```diff
@@ -123,12 +123,9 @@
           center = this.getCenterPoint();
 
       object.set({
-        originalLeft: objectLeft,
-        originalTop: objectTop,
         left: objectLeft - center.x,
         top: objectTop - center.y
       });
-      object.setCoords();
```

Comment:
no reason to make setCoords relative to center of group.
oCoords are used for mouse interactions and object interactions ( collision )